hs_core/templates/pages/my-resources.html

```diff
@@ -56,15 +56,19 @@ <h2 class="page-title">Resources</h2>
             </table>
 
             <div class="btn-toolbar">
-                <a class="btn btn-link" href="?start=0"><i class="fa fa-fast-backward"></i></a>
-                <a class="btn btn-link" href="?start={{ first|add:"-20" }}"><i class="fa fa-backward"></i></a>
+                <a {% if first == 0 %} disabled {% endif %}
```

Comment:
The minimum value for first has to be 0. Otherwise, we will end up seeing something like 'Results -2-20 of 95'